net/nanomq/Makefile

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2025 Andrew Yong <me@ndoo.sg>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nanomq
+PKG_VERSION:=0.24.5
+PKG_RELEASE:=1
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.txt
+PKG_MAINTAINER:=Andrew Yong <me@ndoo.sg>
+
+NANONNG_VERSION:=0.24.5
+NANONNG_SOURCE:=nng-$(NANONNG_VERSION).tar.gz?
+NANONNG_SOURCE_URL:=https://codeload.github.com/nanomq/NanoNNG/tar.gz/
+NANONNG_HASH:=b5ebc85864d4c06eeef107240a5c65735cc0529030caacdf102b37da06862792
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/nanomq/nanomq/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=4992b45082f7d4b859ec2f5d45cb491303fb1e29ec420a03b9fa7256de862f9f
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_OPTIONS+= \
+	-DBUILD_CLIENT=OFF
+
+define Package/nanomq
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=NanoMQ Broker
+  URL:=https://github.com/nanomq/nanomq
+  DEPENDS:=+libatomic +libpthread +librt
```

Comment:
`libpthread` and `librt` are integrated into the libc already, please remove them.